index.html

```diff
@@ -273,6 +276,53 @@ <h1 class="section-title">
             <!-- PAGE LOCATION -->
             <!-- /PAGE LOCATION -->
 
+            <!-- PAGE CODE OF CONDUCT -->
+            <section class="page-section" id="coc">
+                <div class="container">
+                    <h1 class="section-title">
+                        <span data-animation="flipInY" data-animation-delay="300" class="icon-inner"><span class="fa-stack"><i class="fa rhex fa-stack-2x"></i><i class="fa fa-star fa-stack-1x"></i></span></span>
+                        <span data-animation="fadeInRight" data-animation-delay="500" class="title-inner">Code of Conduct</span>
+                    </h1>
+                    <div class="row">
+                        <div class="col-lg-12">
+                            <p data-animation="fadeInUp" data-animation-delay="300">
```

Comment:
I can't find where this initial **p** tag is closed. Can you please double check?